addons/binding/org.openhab.binding.oceanic/README.md

```diff
@@ -76,3 +76,29 @@ Number oceanicConsComplete "volume all time is [%d]"(oceanic) {channel="oceanic:
 Number oceanicConsUntreated "volume untreated is [%d]"(oceanic) {channel="oceanic:softener:s1:consumptionuntreated"}
 Number oceanicConsLastWk "volume last week is [%d]"(oceanic) {channel="oceanic:softener:s1:consumptionlastweek"}
 ```
+
+## Note about nrjavaserial
+
+The Oceanic binding makes use of the nrjavaserial library. There is a known issue (https://github.com/NeuronRobotics/nrjavaserial/issues/96) that requires a workaround on some types of systems
+
+On Ubuntu 17.10 nrjavaserial seems to return only HEX 00 characters through the InputStream of the SerialPort. The solution is to implement a workaround with socat and pipe the data from the Serial Port to a pseudo tty, which has to be manipulated in a CommPortIdentifier.PORT_RAW manner.
+
+             /usr/bin/socat -v /dev/ttyUSB0,raw,echo=0 pty,link=/dev/ttyS1,raw,echo=0         
+The workaround can be implemented using a systemd system manager script, for example:
+
+             [Install]
+             WantedBy=multi-user.target   
+              
+             [Service]
+             #Type=forking
+             ExecStart=/usr/bin/socat -v /dev/ttyUSB0,raw,echo=0 pty,link=/dev/ttyS1,raw,echo=0
+             #PIDFile=/var/run/socat.pid
+             User=root
+             Restart=always
+             RestartSec=10             
+However, in order to fix permissions at the OS level, one has to issue following commands
+
+             sudo useradd  -G dialout openhab
+             sudo chgrp dialout /dev/ttyS1
+             sudo chmod 777 /dev/ttyS1        
+in order to make /dev/ttyS1 accessible by the 'openhab' system user (that is used to start up the openHAB runtime), and to make the tty both readable and writable. Alternatively, these commands can be executed through a script that is attached to the systemd system manager script
```

Comment:
Maybe add a newline in front of this

Reply:
@martinvw Ah.. I removed the newline because when doing the preview the editor showed some additional greyed boxing around those lines, and I thought that was not beautiful. I will change if you like

Reply:
Hmm, I know that rendering sometimes fails because of missing new lines. Maybe use a quoted code block next time.